scripts/dev/release.sh

```diff
@@ -65,7 +69,7 @@ mkdir -p /var/www/versions/v${version}
 mv *.tar.gz /var/www/versions/v${version}
 mv *.zip /var/www/versions/v${version}
 
-wget http://openframeworks.cc/release_hook.php?version=${version} -O /dev/null
+downloader http://openframeworks.cc/release_hook.php?version=${version}
```

Comment:
This will leave temporary files around that we don't want, -O (for output to a file) should be an option of the downloader script